README.md

```diff
@@ -10,6 +10,8 @@
 <p align="center">
 <!-- Latest release -->
 <img src="https://img.shields.io/github/v/release/AndroidIDEOfficial/AndroidIDE?include_prereleases&amp;label=latest%20release" alt="Latest release">
+<!-- Repo Size -->
+<img src="https://img.shields.io/github/repo-size/itsaky/AndroidIDE" alt="Repo Size">

```

Comment:
Update the URL and it at the lasy position in the badges.

Also, create a new PR to the `dev` branch instead of the `main` branch.